terraform/datadog/aws.tf

```diff
@@ -58,3 +58,34 @@ resource "datadog_monitor" "rds-failure" {
 
   tags = ["deployment:${var.env}", "service:${var.env}_monitors", "job:all"]
 }
+
+resource "datadog_monitor" "ec2-cpu-utilisation-warning" {
+  name                = "${format("%s CPU load has been moderately high for a long time on {{bosh-job.name}}/{{bosh-index.name}}", var.env)}"
```

Comment:
Can we use "CPU utilization" here because talking about "load" is confusing.

Reply:
I changed load to utilisation.